lms/djangoapps/teams/tests/test_views.py

```diff
@@ -709,6 +709,30 @@ def test_access_by_username(self, user, status, has_content):
             else:
                 self.assertEqual(membership['count'], 0)
 
+    def test_username_course_filter(self):
```

Comment:
Nitpick: since these two test cases primarily test filtering by `course_id`, I'd recommend emphasizing that in the test case name by placing it first. For example, `test_course_id_with_username` and `test_course_id_with_team_id`.